.travis.yml

```diff
@@ -13,10 +13,22 @@ cache:
 
 matrix:
   include:
+    - name: iOS Debug
```

Comment:
The static checks should be the very first in the matrix so that if a PR has style errors you receive them as fast as possible

Reply:
Hi there, I reverted https://github.com/GodotNativeTools/godot-cpp/pull/298/commits/e33177d96df84278e9a97da279fe8047ee09a08c as it does not have any effect on ci build time.